servers/rendering/rendering_device.h

```diff
@@ -796,8 +813,8 @@ class RenderingDevice : public Object {
 			}
 		};
 
-		StencilOperationState stencil_operation_front;
-		StencilOperationState stencil_operation_back;
+		StencilOperationState front_op;
+		StencilOperationState back_op;
```

Comment:
Is there a particular reason for this rename. It seems worse than the original name to me.

Reply:
yeah, the binder exposes the whole thing as a sub-variable, and the name ended up being too long. Also stencil_ is redundant in this context.